test/parallel/test-inspector-promises.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+const common = require('../common');
+common.skipIfInspectorDisabled();
+
+const assert = require('assert');
+const inspector = require('inspector/promises');
+
+const {
+  basename
+} = require('path');
+const currentFilename = basename(__filename);
+
+{
+  // Ensure that inspector/promises has the same signature as inspector
+  assert.deepStrictEqual(Reflect.ownKeys(inspector), Reflect.ownKeys(require('inspector')));
+}
+
+(async () => {
+  {
+    // Ensure that session.post returns a valid promisified result
+    const session = new inspector.Session();
+    session.connect();
+
+    await session.post('Profiler.enable');
```

Comment:
Could we also add some tests with just plain Promises and some modern Promise utilities such as `Promise.all, allSettled` just curious to cover these scenarios to create an expectancy of how it should behave. (I honestly don't think they wouldn't be respected, but sometimes better have more tests than lesser ;D)